app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -20,7 +20,9 @@ def initialize(cfg = {})
       'hawkular_datasource_remove.ok',
       'hawkular_deployment.ok',
       'hawkular_deployment_remove.ok',
-      'hawkular_event' # # general purpose detail level event
+      'hawkular_event', # general purpose detail level event
+      # filtered (not shown in timeline)
```

Comment:
@jshaughn - if I understand the purpose of this PR correctly,  it looks like the relevant event_type is now 'hawkular_alert' and no longer 'hawkular_event'. If that is the case, should 'hawkular event' be deleted from the white list?